sdk/lib/convert/html_escape.dart

```diff
@@ -137,6 +154,16 @@ class HtmlEscapeMode {
 /// Escaping `>` (greater than) isn't necessary, but the result is often
 /// found to be easier to read if greater-than is also escaped whenever
 /// less-than is.
+///
+/// Example:
+/// ```dart
+/// const sample = 'Text & subject, <, >, "quoted", \'single-quoted\', /';
+///
+/// final htmlEscaped = const HtmlEscape().convert(sample);
+/// print(htmlEscaped);
+/// // Text &amp; subject, &lt;, &gt;, &quot;quoted&quot;,
+/// // &#39;single-quoted&#39;, &#47;
```

Comment:
Could we show that this is the output a bit more clearly? 

I found it confusing, especially reading it in the source code as you have `/// //`